akka-actor-typed/src/main/scala/akka/actor/typed/Scheduler.scala

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (C) 2009-2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.actor.typed
+
+import akka.actor.Cancellable
+import akka.annotation.DoNotInherit
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.duration.FiniteDuration
+
+/**
+ * The ActorSystem facility for scheduling tasks.
+ *
+ * For user scheduling needs `Behaviors.withTimers` should be preferred.
```

Comment:
I think it's more like "For scheduling within actors`Behaviors.withTimers` should be preferred."